scripts/zap-scanner.sh

```diff
@@ -5,23 +5,16 @@ set -o pipefail
 
 TARGET=$1
```

Comment:
@n0remac i finally figured out what was going on with the passive rules not being skipped on our prod acf urls. To summarize:
- target = the full url to be scanned in `zap-hook.py`, whereas 
- target = 'frontend' or 'backend' in `zap-scanner.sh` 
- I added to the if blocks in `zap-hook.py`, and these scans now give what is expected. 
  - successful scans [here](https://app.circleci.com/pipelines/github/raft-tech/TANF-app/9754/workflows/a3261280-12b5-4ca5-9f5c-7c61bc2a2510)
  - my test branch with relevant changes in the abovementioned scripts [here](https://github.com/raft-tech/TANF-app/tree/qasp/1754-owasp-on-additional-branches)